x-pack/plugins/observability/server/services/slo/update_slo.ts

```diff
@@ -87,6 +89,7 @@ export class UpdateSLO {
     });
 
     await this.deleteOriginalSLO(originalSlo);
+    await this.systemClient.enrich.executePolicy({ name: SLO_SUMMARY_ENRICH_POLICY_NAME });
```

Comment:
We should do this sooner but **after** the repository save I think. For example, if the slo name or description changes, we won't reach here, but we do update the SLO in the repository.